### ISSUE TITLE
[Bug Report] Menu is closed when selecting text and mouse up happens outside menu

### ISSUE THREAD

**Issue by StefanFabian** [vuetifyjs/vuetify#5886]
### Versions and Environment
**Vuetify:** 1.3.14
**Vue:** 2.5.21
**Browsers:** Firefox 64.0
**OS:** Windows 10

### Steps to reproduce
Click the link icon.
Start a selection and drag the mouse outside of the menu before letting go.

### Expected Behavior
The text should be selected.

### Actual Behavior
The menu closes.

### Reproduction Link
<a href="https://codepen.io/anon/pen/jXWbjz" target="_blank">https://codepen.io/anon/pen/jXWbjz</a>

### Other comments
To be honest, I would generally expect mouse downs inside the menu to not close it if close-on-content-click is false even if the mouse up is outside.

PS: I really like the simplicity and elegance of vuetify, so, good job!

**Comment by johnleider** [comment]
Need additional steps/information. What I'm reproducing works so I figure it's not your issue.

**Comment by MajesticPotatoe** [comment]
Not reproducible in chrome or edge. Firefox confirmed.

**Comment by bom2ne1** [comment]
I managed to fix this by making some changes to the click-outside directive.

I added a boolean to flag whether mousedown began on the target el.

```
const onMouseDown = (e: Event) => {
      if (el.contains(e.target)) {
          el._mouseDownOnEl = true;
      }
    }
```

then return early if true:

```
  if (typeof el._mouseDownOnEl !== 'undefined' && el._mouseDownOnEl) {
      el._mouseDownOnEl = false;
      return
  }
```

idk if this'll break anything else, I've not really tested it's interaction with other components., but it worked in my case so I'll use it lol.

If someone wants to test it, and maybe pr it, here's a pastebin of the click-outside.ts file I'm using:
https://pastebin.com/Y611G3kC
